src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { FilterTodosBy } from '../../types/FilterTodosBy';
+
+type Props = {
+  setFilterBy: (FilterTodosBy: FilterTodosBy) => void,
+  filterBy: string,
+};
+export const TodoFilter: FC<Props> = ({
+  setFilterBy,
+  filterBy,
+}) => {
+  const filteredByOptions = Object.entries(FilterTodosBy);
+
+  return (
+    <nav className="filter">
+
+      {filteredByOptions.map(([label, value]) => (
+        <a
+          href={`#/${value}`}
+          className={classNames(
+            'filter__link',
+            { selected: filterBy === value },
+          )}
+          key={value}
+          onClick={() => setFilterBy(value)}
```

Comment:
Type of 'value' should be explicitly set to 'FilterTodosBy' in the 'setFilterBy' function call.